apps/site/assets/ts/models/alert.ts

```diff
@@ -19,6 +20,92 @@ export const alertsByStop = (alerts: Alert[], stopId: StopId): Alert[] =>
       !!entities.stop && entities.stop!.some((id: StopId) => id === stopId)
   );
 
+const hasEffect = (alerts: Alert[], effect: string): boolean =>
+  some(alerts, alert => alert.effect === effect);
+
+export const isSuspension = (alerts: Alert[]): boolean =>
```

Comment:
nitpick(non-blocking): Consider renaming these `hasX` instead of `isX` since it is looking in a list of alerts